meinberlin/assets/scss/components_user_facing/_moderator_notes.scss

```diff
@@ -0,0 +1,73 @@
+.moderator-notes {
+    background-color: $bg-secondary;
+    padding: $spacer;
+}
+
+.moderator-notes__summary {
+    display: flex;
+    justify-content: space-between;
+
+    &::marker {
+        display: none;
+        content: "";
+    }
+
+    span {
+        display: flex;
+
+        &:before {
+            font-size: 22px;
+            align-self: center;
+        }
+    }
+}
+
+.moderator-notes__details[open] .moderator-notes__summary>span:before {
+    transform: rotate(180deg);
```

Comment:
more of a question: we have general css for the detail accordion here https://github.com/liqd/a4-meinberlin/blob/dev/meinberlin/assets/scss/components/_details.scss#L18 and now more specific one here, wouldn't it be easier to just add the `span` along with the `i`instead of duplicating it?